R2API/UnbundledResourcesProvider.cs

```diff
@@ -101,7 +122,9 @@ public sealed class UnbundledResourcesProvider : IResourceProvider {
         }
 
         public UnityObject[] LoadAll(Type type) {
-            return resources.Values.Where((obj) => obj.GetType() == type || obj.GetType().IsSubclassOf(type)).ToArray();
+            return typedResources
+                .Where(kv => kv.Key == type || kv.Key.IsSubclassOf(type))
```

Comment:
Would Type.IsAssignableFrom be more permissive here and still work properly for the desired cases?

Reply:
Ah, yes it would. Made the change.